src/mutate/deleteNode.ts

```diff
@@ -96,7 +96,7 @@ export function deleteOrSetNodeDeleted(
     return
   }
   const newDeleted = deleted
-    ? { ...deleted, updatedAt: deleteAttrs.updatedAt }
+    ? { ...deleted, updatedAt: deleteAttrs.updatedAt, status: deleted.status === CHANGE_STATUS.rejected ? CHANGE_STATUS.pending : deleted.status }
```

Comment:
Would that make sense to actually create a new deleted instead of reusing already existing to track it as a separate action?
My concern here is with the cases where we might delete something and it would bring back something what was already rejected and will (surprisingly for user) require rejecting again?

Reply:
yes that makes sense, updated.